homeassistant/components/light/yeelight.py

```diff
@@ -178,7 +178,7 @@ def effect_list(self):
     @property
     def unique_id(self) -> str:
         """Return the ID of this light."""
-        return "{}.{}".format(self.__class__, self._ipaddr)
+        return self._ipaddr
```

Comment:
IP address. These devices would have two ways to obtain a unique identifier (either by using miio's proprietary api, a bit no-go as the current component uses a different one) or via mdns (`mac` field).

Reply:
Yeah I agree, IP address is not a good thing to see as uniqueness. Will remove them.